pkg/controllers/mcs/service_import_controller.go

```diff
@@ -153,24 +154,15 @@ func (c *ServiceImportController) updateServiceStatus(svcImport *mcsv1alpha1.Ser
 	}
 
 	err := retry.RetryOnConflict(retry.DefaultRetry, func() (err error) {
-		derivedService.Status = corev1.ServiceStatus{
```

Comment:
The similar code has not been fully modified yet, for example https://github.com/karmada-io/karmada/blob/master/pkg/controllers/multiclusterservice/endpointslice_dispatch_controller.go#L120-L143

I suggest you do a global search for `RetryOnConflict` and double-check it.

Reply:
done